app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -102,3 +101,95 @@ th .sort-arrow {
     vertical-align: middle;
     color: #fff;
 }
+
+.table-filters {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 100%;
+    padding: 10px 0;
+    margin-bottom: 10px;
+}
+
+.filter-controls {
+    display: flex;
+    align-items: center;
+    gap: 20px;
+}
+
+.orcid-switch {
+    display: flex;
+    align-items: center;
+}
+
+.orcid-toggle-label {
+    margin-right: 10px;
+    font-size: 14px;
+    color: #333;
+    white-space: nowrap;
+    font-weight: lighter;
+}
+
+.switch {
+    position: relative;
+    display: inline-block;
+    width: 60px;
+    height: 30px;
+}
+
+.switch input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+    display: none;
```

Comment:
I believe doing `display: none` on an input will make it impossible to tab to, so I'd like to remove this property for accessibility sake.